### PR TITLE
Zonal elevation 

### PR DESCRIPTION
**Pull request by dgboss**
Capture some basic elevation stats for each firezone where HFI values exceed the advisory and warning thresholds including:
- min
- max
- mean
- 25th quartile
- 75th quartile

To run this locally you will need to add a key/value to your api .env:

DEM_NAME=dem_mosaic_250_max.tif

You will also need to apply migrations.
# Test Links:
[Percentile Calculator](https://wps-pr-2569.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-2569.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-2569.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-2569.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-2569.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-2569.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-2569.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

File: api/app/auto_spatial_advisory/run_type.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+"""A class representing HFI run types: actual or forecast"""
```
Comment (conbrad) [inline api/app/auto_spatial_advisory/run_type.py:1]: Thanks for pulling this out!

**Review by conbrad — APPROVED** [review]
Nice 🚢

File: api/app/db/models/auto_spatial_advisory.py [hunk 2]
```diff
@@ -137,3 +137,25 @@ class RunParameters(Base):
...
+    )
+    id = Column(Integer, primary_key=True, index=True)
+    advisory_shape_id = Column(Integer, ForeignKey('advisory_shapes.id'), nullable=False, index=True)
+    threshold = Column(Integer, ForeignKey('advisory_hfi_classification_threshold.id'), nullable=False)
+    run_parameters = Column(Integer, ForeignKey('run_parameters.id'), nullable=False, index=True)
+    minimum = Column(Float, nullable=False)
+    quartile_25 = Column(Float, nullable=False)
+    median = Column(Float, nullable=False)
+    quartile_75 = Column(Float, nullable=False)
+    maximum = Column(Float, nullable=False)
```
Comment (andrea-williams) [inline api/app/db/models/auto_spatial_advisory.py:161]: shouldn't there also be a column here for `median`? The median value is calculated in auto_spatial_advisory/elevation.py

**Review by andrea-williams — APPROVED** [review]

File: api/app/db/models/auto_spatial_advisory.py (hunk 2 above)
Comment (dgboss) [reply api/app/db/models/auto_spatial_advisory.py:161]: It's hiding on line 159!

**Pull request closed without merging**